dev/source/docs/ardupilot-on-linux-init-scripts.rst

```diff
@@ -0,0 +1,38 @@
+.. _ardupilot-on-linux-init-scripts:
+
+============================
+Linux Initialization Scripts
+============================
+
+
+Example Systemd Unit File
+=========================
+
+Adjust the following and place in /etc/systemd/system/arduplane.service
+
+::
+
+    [Unit]
+    Description=ArduPlane Service
+    Conflicts=arducopter.service ardupilot.service ardurover.service
+
+    [Service]
+    EnvironmentFile=/etc/default/arduplane
+
+    # Lock CPU governor to top speed for reduced latency
+    ExecStartPre=/bin/bash -c "/bin/echo performance | tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor"
```

Comment:
as /bin/bash is used, we probably need to specify this example is for a Debian/ubuntu distribution, just to be safe for those that don't have bash

Reply:
I think everything should work with /bin/sh, I'll change it to do so.